Content.Shared/_CD/CartridgeLoader/Cartridges/VoteLinkUiState.cs

```diff
@@ -0,0 +1,99 @@
+using Robust.Shared.Serialization;
+using Robust.Shared.Timing;
+
+namespace Content.Shared._CD.CartridgeLoader.Cartridges;
+
+[Serializable] [NetSerializable]
+public sealed class VoteLinkUiState(
+    VoteData? activeVote,
+    List<VoteData> history,
+    TimeSpan? cooldown = null,
+    bool hasAccess = false)
```

Comment:
why just check access on the client

Reply:
how the fuck did I miss this :(

Reply:
i already forgot how this code works, saw this and went "what why the fuck"

Reply:
That is what you get for coding while sleep-deprived. If I remember correctly I also reviewed this at midnight.